gui/src/2D/controls/inputTextArea.ts

```diff
@@ -0,0 +1,1282 @@
+import { Nullable } from "babylonjs/types";
+import { Observable } from "babylonjs/Misc/observable";
+import { Vector2 } from "babylonjs/Maths/math.vector";
+
+import { Control } from "./control";
+import { ValueAndUnit } from "../valueAndUnit";
+import { VirtualKeyboard } from "./virtualKeyboard";
+import { RegisterClass } from 'babylonjs/Misc/typeStore';
+import { Measure } from '../measure';
+import { InputText } from "./inputText";
+import { ICanvasRenderingContext, ITextMetrics } from "babylonjs/Engines/ICanvas";
+import { PointerInfoBase } from "babylonjs/Events/pointerEvents";
+import { IKeyboardEvent } from "babylonjs/Events/deviceInputEvents";
+
+/**
+ * Enum that determines the text-wrapping mode to use.
+ */
+export enum TextWrapping_ {
+    /**
+     * Clip the text when it's larger than Control.width; this is the default mode.
+     */
+    Clip = 0,
+
+    /**
+     * Wrap the text word-wise, i.e. try to add line-breaks at word boundary to fit within Control.width.
+     */
+    WordWrap = 1,
+
+    /**
+     * Ellipsize the text, i.e. shrink with trailing … when text is larger than Control.width.
+     */
+    Ellipsis,
+}
+
+/**
+ * Class used to create input text control
+ */
+export class InputTextArea extends InputText {
+
+    //private _textWrapping = TextWrapping_.Clip;
```

Comment:
would be great to cleanup any unneeded / commented line of code

Reply:
Removed!